packages/augur-tools/src/flash/scripts.ts

```diff
@@ -1834,52 +1804,31 @@ export function addScripts(flash: FlashSession) {
         required: false,
       },
       {
-        name: 'isInvalid',
-        abbr: 'i',
-        description:
-          'isInvalid flag is used only for scalar markets (optional)',
-        required: false,
+        name: 'payoutNumerators',
+        abbr: 'p',
+        description: 'payout numerators of child unverse.',
+        required: true,
       },
     ],
     async call(this: FlashSession, args: FlashArguments) {
       if (this.noProvider()) return;
-      const user = await this.ensureUser();
+      this.config.gnosis.enabled = false;
```

Comment:
This will fail if `config.gnosis` is undefined. Use `mergeConfig`:

```suggestion
      this.config = mergeConfig(this.config, { gnosis: { enabled: false }});
```